core/includes/bootstrap.inc

```diff
@@ -2504,14 +2537,16 @@ function backdrop_set_message($message = NULL, $type = 'status', $repeat = TRUE)
 /**
  * Returns all messages that have been set with backdrop_set_message().
  *
- * @param string $type
+ * @param string|null $type
  *   (optional) Limit the messages returned by type. Defaults to NULL, meaning
  *   all types. These values are supported:
+ *   phpcs:disable
  *   - NULL
  *   - 'status'
  *   - 'warning'
  *   - 'error'
  *   - 'info'
+ *   phpcs:enable
```

Comment:
An alternative could be to add the full stop phpcs requests. Only one at the end of the last line.

I know, that's actually silly (in our standard check). But AFAIK these `phpcs:enable` will show up literally in our documentation, which might be more confusing than a single dot.